apps/crn-frontend/src/AuthenticatedApp.tsx

```diff
@@ -95,6 +100,11 @@ const AuthenticatedApp: FC<Record<string, never>> = () => {
                   <Discover />
                 </Frame>
               </Route>
+              <Route path={tutorials.template}>
```

Comment:
To get coverage for the tutorials route you would need to add a test to this file that navigates there however I don't think this is in the right place.

I expected the route to be /discover/tutorials/:id which would follow the pattern set in Network.

Reply:
Thanks! I didn't understand before, it's committed here the change to make the route of turorial details page to be  `/discover/tutorials/:id ` https://github.com/yldio/asap-hub/pull/2143/commits/d27598d2611210f076b161f5bd49f9bc61e275b9